src/assignments/mars-rover.rst

```diff
@@ -185,21 +267,102 @@ several functions outside of ``constructor``.
 #. ``receiveMessage(message)``
 
    a. ``message`` is a ``Message`` object
-   b. Returns an object containing two properties---the original message and an
-      array of *results*. Each element in the array is an object that
-      corresponds to one ``Command`` in ``message.commands``.
-   c. Specific details about how to respond to different commands are in the
-      :ref:`Test List <rover-test-list>`.
+   b. Returns an object containing at least two properties:
+         
+      i. ``message``: the name of the original ``Message`` object
+      ii. ``results``: an array of *results*. Each element in the array is an 
+          object that corresponds to one ``Command`` in ``message.commands``.
+         
+   c. Updates certain properties of the rover object
+
+      i. Details about how to respond to different commands are in the
+         :ref:`Command Types table <command-types-table>`.
 
 .. admonition:: Example
 
    .. sourcecode:: js
 
       let commands = [new Command('MODE_CHANGE', 'LOW_POWER'), new Command('STATUS_CHECK')];
-      let message = new Message('e1', commands);
-      let rover = new Rover(98382);
+      let message = new Message('Test message with two commands', commands);
+      let rover = new Rover(98382);    // Passes 98382 as the rover's position.
       let response = rover.receiveMessage(message);
 
+      console.log(response);
+
+   **Output**
+
+   ::
+
+      {
+         message: 'Test message with two commands',
+         results: [
+            {
+               completed: true
+            },
+            {
+               completed: true, 
+               roverStatus: {
+                  mode: 'LOW_POWER', 
+                  generatorWatts: 110, 
+                  position: 98382
+               }
+            }
+         ]
+      }
```

Comment:
How about condensing the response objects into single lines?  Something like
{ completed: true }, 
{ completed: true, roverStatus: {...} }

Spreading the objects out makes the output less clear (too much whitespace).